### DISCUSSION TITLE
IIS Log Analyzer (offline)

### DISCUSSION THREAD

**Discussion by hsninbil** [Security-Onion-Solutions/securityonion#5597] [Unsupported Versions] [answered]
Dear,
i want to analyze IIS log files (offline) with Security Onion?
Thank You

**Comment by Sahale** [comment, marked as answer]
I don't understand what means offline, but maybe you want to use IIS [filebeat module](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-iis.html)?
